client/components/date-range/index.js

```diff
@@ -649,9 +655,17 @@ export class DateRange extends Component {
 			'date-range__picker': true,
 		};
 
+		const calendarInitialDate =
```

Comment:
Definitely not intending to block the PR or anything, but in general, I'd always recommend keeping a fix separate from the test migration. Or perhaps there is a good reason to keep them together?

Reply:
In this case, the logic here was needed for some of the migrated tests to pass. More specifically the ones that test the `firstSelectableDate` or `lastSelectableDate`. Before this fix, setting the `firstSelectableDate` would not render the calendars in the correct range, it'd still use today's date to calculate the range. 

The previous non-RTL test was passing because it was checking the state directly (if my memory doesn't fail), but RTL tests require the actual elements to be visible so that we can interact with them.

Reply:
Sounds good. We discussed that in our last pairing session, but I just wanted to have it written somewhere. 😉